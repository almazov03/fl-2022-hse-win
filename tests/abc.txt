STARTING_NON_TERMINAL = $S$;
$S$ = { "a" $S$ }; $S$ = { "b" $S$ };
$S$ = { "c" $S$ }; $S$ = { EPS };
