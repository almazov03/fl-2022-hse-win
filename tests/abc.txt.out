LexToken(START,'STARTING_NON_TERMINAL',1,0)
LexToken(EQUAL,'=',1,22)
LexToken(NON_TERMINAL,'S',1,24)
LexToken(SEPARATOR,';',1,27)
LexToken(NON_TERMINAL,'S',2,30)
LexToken(EQUAL,'=',2,34)
LexToken(LBR,'{',2,36)
LexToken(TERMINAL,'a',2,38)
LexToken(NON_TERMINAL,'S',2,42)
LexToken(RBR,'}',2,46)
LexToken(SEPARATOR,';',2,47)
LexToken(NON_TERMINAL,'S',2,49)
LexToken(EQUAL,'=',2,53)
LexToken(LBR,'{',2,55)
LexToken(TERMINAL,'b',2,57)
LexToken(NON_TERMINAL,'S',2,61)
LexToken(RBR,'}',2,65)
LexToken(SEPARATOR,';',2,66)
LexToken(NON_TERMINAL,'S',3,69)
LexToken(EQUAL,'=',3,73)
LexToken(LBR,'{',3,75)
LexToken(TERMINAL,'c',3,77)
LexToken(NON_TERMINAL,'S',3,81)
LexToken(RBR,'}',3,85)
LexToken(SEPARATOR,';',3,86)
LexToken(NON_TERMINAL,'S',3,88)
LexToken(EQUAL,'=',3,92)
LexToken(LBR,'{',3,94)
LexToken(EMPTY,'EPS',3,96)
LexToken(RBR,'}',3,100)
LexToken(SEPARATOR,';',3,101)
